module/mempool/stdmap/backDataHeapBenchmark_test.go

```diff
@@ -193,14 +193,47 @@ func (b baselineLRU) All() map[flow.Identifier]flow.Entity {
 
 		entity, ok := b.ByID(id)
 		if !ok {
-			panic("could not retrive entity from mempool")
+			panic("could not retrieve entity from mempool")
 		}
 		all[id] = entity
 	}
 
 	return all
 }
 
+func (b baselineLRU) Identifiers() flow.IdentifierList {
+	ids := make(flow.IdentifierList, b.c.Len())
+	i := 0
+	for _, entityID := range b.c.Keys() {
```

Comment:
since the 'range' keyword is causing the compiler to generate a for loop, and since 'Keys()' is an array type, it would be preferable to use 'i' like this:  'for i:=0;i < keyArrayLen;i++' and avoid using the 'range' keyword entirely.  That cuts the computational work in half.

Reply:
https://github.com/onflow/flow-go/pull/1852/commits/f2768318602b479b6a3ad0f9c96db6370433310b